learningresources/tests/base.py

```diff
@@ -61,17 +66,19 @@ def create_resource(self, **kwargs):
         """Creates a learning resource with extra fields"""
         learn_res = create_resource(
             course=self.course,
-            parent=None,
-            resource_type=kwargs.get('resource_type') or "example",
-            title=kwargs.get('title') or "other silly example",
-            content_xml=kwargs.get('content_xml') or "<blah>other blah</blah>",
-            mpath=kwargs.get('mpath') or "/otherblah",
-            url_name=kwargs.get('url_name') or None
+            parent=kwargs.get('parent'),
+            resource_type=kwargs.get('resource_type', "example"),
+            title=kwargs.get('title', "other silly example"),
+            content_xml=kwargs.get('content_xml', "<blah>other blah</blah>"),
+            mpath=kwargs.get('mpath', "/otherblah"),
+            url_name=kwargs.get('url_name'),
+            dpath=''
         )
         learn_res.xa_nr_views = kwargs.get('xa_nr_views', 0)
         learn_res.xa_nr_attempts = kwargs.get('xa_nr_attempts', 0)
         learn_res.xa_avg_grade = kwargs.get('xa_avg_grade', 0)
         learn_res.save()
+        update_description_path(learn_res)
```

Comment:
I think this means we have two saves happening unnecessarily. No strong opinion on this though, it's probably not often called